.github/gbp.toml

```diff
@@ -14,6 +14,8 @@ reset_label = "GBP: Reset"
 "Priority: CRITICAL" = 20
 "Priority: High" = 15
 "Quality of Life" = 1
+"Content" = -5
+"Jobtent" = 5
```

Comment:
this is just going to reward low quality feature coding by people that just make shit without  thinking about if it actually improves the game or not and will just make the entire gbp system useless.

Reply:
I take it that this is in reference to the jobtent tag since its positive.
If something is low quality and doesn't improve the game I don't think it should merged. If maintainers consider low quality rushed pointless features to add interesting content and depth to some more shallow job features then this would be an issue. But I trust that the maintainer team would not merge stuff of this type.

Reply:
it's still just feature coding and definitely isn't worth the same as half a refactor. This comes across as a "I don't have GBP but dont want to fix bugs or code" PR.

Reply:
Sorry if it seems this way, I didn't intend for that to be a motivating aspect of this. My two main intentions was this to discourage people from making giant gameplay changes over improving other people's gameplay changes. Encourage people to expand upon game loops that are currently quite shallow.
To me a good job gameplay loop is one that contains depth, a learning curve and variety. It should take a decent amount of time to "finish a job", you should become better at the job the more you play it, and it should take several rounds to see everything a job has to offer. I give xenobiology and shaft mining as decent examples of this. Both are things that cannot be "completed" in an average round, and both have a clear learning curve, and both can be played in very different ways to add variety. (Different starting weapons for shaft miner (+ ruin RNG) (Xenobiology has some many things you can make with it, its absurd to try and do them all in one round, so people focus certain subsets of items that they want per round) I feel like a lot of recent gameplay changes have not offered both depth, a sufficient learning curve and variety. "content" expansions would add more stepping stones for learning, and more total options to increase variety and replayabilty of a job